searx/utils.py

```diff
@@ -800,3 +803,12 @@ def js_variable_to_python(js_variable):
     s = s.replace(chr(1), ':')
     # load the JSON and return the result
     return json.loads(s)
+
+
+def render_template(template_path, **args):
+    templates_path = settings['ui']['templates_path']
+    with open(os.path.join(templates_path, "simple/" + template_path)) as f:
```

Comment:
The `simple` theme is just hardcoded here, but we should use the theme from the request cookies - not sure how to get that inside a search engine module though.